yktracec/src/jitmodbuilder.cc

```diff
@@ -1811,10 +1752,12 @@ class JITModBuilder {
                 assert(cast<GetElementPtrInst>(I)->getPointerOperand() ==
                        TraceInputs);
                 I++;
+                CurInstrIdx++;
```

Comment:
Why all the `CurInstrIdx++`?

Reply:
I need them for the `insertAOTMap` further below.